markdown.dtx

```diff
@@ -4985,6 +4985,52 @@ defaultOptions.inlineFootnotes = false
 %</lua,lua-cli>
 %<*manual-options>
 
+#### Option `jekyllData`
+
+`jekyllData` (default value: `false`)
+
+% \fi
+% \begin{markdown}
+%
+% \Optitem[false]{jekyllData}{\opt{true}, \opt{false}}
+%
+:    true
+
+     :   Enable the Pandoc `yaml_metadata_block` syntax extension for entering
+         metadata in YAML:
+
+~~~~~~ yaml
+---
+title:  'This is the title: it contains a colon'
+author:
+- Author One
+- Author Two
+keywords: [nothing, nothingness]
+abstract: |
+  This is the abstract.
+
+  It consists of two paragraphs.
+---
+~~~~~~~~~~~
+
+:    false
+
+     :   Disable the Pandoc `yaml_metadata_block` syntax extension for entering
+         metadata in YAML.
+
+% \end{markdown}
+% \iffalse
+%</manual-options>
+%<*lua,lua-cli>
+% \fi
+%  \begin{macrocode}
+defaultOptions.jekyllData = false
+%    \end{macrocode}
+% \par
+% \iffalse
+%</lua,lua-cli>
+%<*manual-options>
+
```

Comment:
This is ok for now, but we will need to add some examples after we're happy with the implementation.

Reply:
Fixed in 5c44dca.